src/components/mixins/player.js

```diff
@@ -888,22 +888,17 @@ export const playerMixin = {
 
     reloadAnnotations(current = true) {
       if (!this.annotations) return
-      const annotations = this.annotations.map(a => ({ ...a }))
-      this.annotations = []
-      setTimeout(() => {
-        this.annotations = annotations
-        if (current) {
-          this.reloadCurrentAnnotation()
-        }
-      }, 200)
+      this.annotations = this.annotations.map(a => ({ ...a }))
```

Comment:
This line seems useless, it's a shallow copy of himself.

Reply:
It's done to force Vue reactivity